nibabies/workflows/bold/registration.py

```diff
@@ -499,13 +497,14 @@ def init_bbreg_wf(use_bbr, bold2t1w_dof, bold2t1w_init, omp_nthreads, name='bbre
             (inputnode, mri_coreg, [('subjects_dir', 'subjects_dir'),
                                     ('subject_id', 'subject_id'),
                                     ('in_file', 'source_file')]),
-            (mri_coreg, bbregister, [('out_lta_file', 'init_reg_file')]),
             (mri_coreg, transforms, [('out_lta_file', 'in2')]),
         ])
 
         # Short-circuit workflow building, use initial registration
         if use_bbr is False:
-            select_transform.inputs.index = 1
+            # TIL having Merge(2) with only `in2` input defined
+            # will cause the output list to contain a single element
+            select_transform.inputs.index = 0
```

Comment:
I think we can drop this altogether here and in L525 (pre-diff), and do (see next comment)...